src/CarrierLogo/CarrierLogo.js

```diff
@@ -0,0 +1,77 @@
+// @flow
+
+import * as React from 'react';
+import { View, Image, PixelRatio } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import StyleSheet from '../PlatformStyleSheet';
+import type { Props } from './CarrierLogoTypes';
+import { SIZE_OPTIONS, CARRIER_TYPE_OPTIONS, BASE_URL } from './consts';
+import { logoSizes } from './styles';
+
+export default function CarrierLogo({
+  size = SIZE_OPTIONS.LARGE,
+  carriers = [],
+}: Props) {
+  const carriersLength = carriers.length;
+  const urlSize =
+    (carriersLength > 1 || size === SIZE_OPTIONS.SMALL ? 16 : 32) *
+    (PixelRatio.get() >= 2 ? 2 : 1);
+
+  return (
+    <View
+      style={[
+        styles.wrapper,
+        carriersLength > 1 ? sizeStyles[SIZE_OPTIONS.LARGE] : sizeStyles[size],
+      ]}
+    >
+      {carriers.slice(0, 4).map((carrierImage, index) => (
+        <Image
+          key={carrierImage.code}
```

Comment:
This might be a problem if duplicate carrier is passed in `carriers` prop.

Reply:
I've been thinking about this. But since it's used this way in `orbit` https://github.com/kiwicom/orbit-components/blob/master/src/CarrierLogo/index.js , I've assumed that it cannot happen.
Do you think there will be case when we want to display same logo multiple times? In this case we need to use something else as a key.
Otherwise, I can at least add some data parsing to remove duplicate carriers (if it's desired behaviour).

Reply:
Just to be sure I've added parser which filters out carriers with the same `code` (which serves as id). Also separated it into helpers and added some test for it.